stack.py

```diff
@@ -57,22 +57,77 @@ def parse_program(code):
     [Instr('push', [1.0])]
     >>> list(parse_program('↔'))
     [Instr('swap')]
+
+    Prefixes are placed before the instruction
+    Some prefixes also have sigils
+    >>> list(parse_program('♯ +'))
+    [Instr('add', [], ['quiet'])]
     """
-    for line in re.split('\n|;', code):
+    split_program = re.split('\n|;', code)
+    label_indexes = get_label_indexes(split_program);
```

Comment:
True, your approach has quite a lot going for it in terms of simplicity, I'll certainly give it that.